L/Libxc/Libxc/build_tarballs.jl

```diff
@@ -37,4 +41,5 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               preferred_gcc_version=v"5", julia_compat="1.6")
+               preferred_gcc_version=v"8",
+               julia_compat="1.8")
```

Comment:
Any reason for restricting the compat bound here? Looks unnecessary

Reply:
No good reason, it might have sneaked in when I was trying various things debugging. I reverted it to the original `julia_compat="1.6"`